### ISSUE TITLE
Post with [code] tags promoted to frontpage loses closing >

### ISSUE THREAD

**Issue by Bopske** [Tinyportal/TinyPortal#964]
Put a code block in a post and promote the post to show on front page: it loses all closing > characters ..

**Comment by dragomano** [comment]
Try to disable line #205 in TinyPortal\Util.php:
```php
                        //$string         = html_entity_decode($string, ENT_QUOTES, 'UTF-8');
```

With this line we have this issue:
![sshot-9](https://github.com/Tinyportal/TinyPortal/assets/229402/ea0fb0a7-6099-4c84-b903-809e2fab8e39)

Without this line we do not have this issue:
![sshot-10](https://github.com/Tinyportal/TinyPortal/assets/229402/825041ab-71c5-4acb-8518-5a0f68fe3d66)

**Comment by Bopske** [comment]
included in #967
